d.html.markdown

```diff
@@ -95,16 +97,14 @@ enum Day {
     Wednesday,
     Thursday,
     Friday,
-    Saturday,
```

Comment:
Trailing commas make diffs nicer, make merges easier, and is allowed explicitly by D and any compiler that doesn't support them is in error.